### DISCUSSION TITLE
Newbie on vulkan & validation layer errors/warnings & recommended hardware?

### DISCUSSION THREAD

**Discussion by phstrauss** [datoviz/datoviz#63] [Q&A]
What does mean the whole lot of validation layer warnings and errors? for example, on my low-end GPUs
(an AMD Ryzen2 linux laptop and an summer 2020 intel macbook air scissor with intel iris plus,
both integrated GPU), `./manage.sh test vislib` return 16 errors/fails on 18 tests. Are those errors vulkan
layer errors tied to the GPU software stack (it seems so) or vklite bugs?

Reading the validation layer messages, it seems to be vulkan layer bugs reported by vklite, but is the state
of affair regarding vulkan hardware/software stacks in such a bad state as to not beeing able to run
datoviz on both my integrated GPU Linux/macOS laptops?

I would need datoviz from the C api for animated real-time 3D surface visualization, for waterfall display
(animated 3D spectrogram in audio).
What is the recommended GPU hardware for running datoviz?

TIA!

**Comment by rossant** [comment]
Datoviz v0.3 has just been released. Please feel free to open a new issue if you get any specific warnings.
